### PR TITLE
Malformed data due to data.classe() return undefined #57 and fix Content-Type issue with utf-8

### PR DESCRIPTION
**Pull request by kefahB**
Hi @EddyVerbruggen 

I have fixed those issues that I facing  on `IOS` only : 
- malformed data related with #56 .. the issue is fixed by replacing `data instanceof NSArray` 
 instead of `data.class().name`  
- Content-Type dose not support `application/json; charset=utf-8` #54

### CODE REVIEW

**Comment by kefahB** [comment]
@EddyVerbruggen 
During implementation with `android` I have facing the error `javax.net.ssl.SSLPeerUnverifiedException` .. my certificate is `Domain Control Validated`, after a long debugging, it seems the problem come from the `Wildcard` support .. I have edited `https.android.ts` and I have testing with my certificate and all test was done.

In order to dealing with a `sub-domaine` we must set `commonName` to `*.domaine.com` other way it will return `SSLPeerUnverifiedException`

```ts
Https.request error javax.net.ssl.SSLPeerUnverifiedException: Hostname demo.lunch-digi-pay.lu not verified:
certificate: sha256/i5fW5TLEfJtKsLBatgI2QctF2EPmxuVrGtVGW418CDM=
JS:     DN: CN=*.lunch-digi-pay.lu,OU=Domain Control Validated
JS:     subjectAltNames: [*.lunch-digi-pay.lu, lunch-digi-pay.lu]
```

File: src/https.android.ts [hunk 1]
```diff
@@ -44,6 +44,9 @@ export function enableSSLPinning(options: Https.HttpsSSLPinningOptions) {
     peer.host = options.host;
     peer.commonName = options.commonName || options.host;
     peer.certificate = certificate;
+    if (options.commonName) {
```
Comment (EddyVerbruggen) [inline src/https.android.ts]: This seems redundant because of line 45 above.

File: src/package.json [hunk 2]
```diff
@@ -43,7 +43,13 @@
       "name": "Robert Laverty",
       "email": "roblav96@gmail.com",
       "url": "https://github.com/roblav96"
+    },
```
Comment (EddyVerbruggen) [inline src/package.json:46]: 👍

**Comment by EddyVerbruggen** [comment]
Thanks! I'll merge this asap.

**Pull request merged**